sources.go

```diff
@@ -83,6 +83,31 @@ func ListFiles(path string) *Pipe {
 	return Slice(fileNames)
 }
 
+// FindFiles creates a pipe containing the files inside a directory and all its subdirectories
+// matching the supplied path, one per line.
+func FindFiles(path string) *Pipe {
+
```

Comment:
It's house style in this repo not to use blank lines within functions—or, at least, to do so incredibly sparingly. The trouble with blank lines is, where do you stop? Eventually every other line is blank and the code is twice as long!